src/components/va-accordion/va-accordion-item.tsx

```diff
@@ -25,16 +25,32 @@ export class VaAccordionItem {
    */
   @Prop() open: boolean = false;
 
+  /**
+   * Header level for button wrapper. Must be between 1 and 6
+   */
+  @Prop() level: number = 2;
+
   private toggleOpen(e: MouseEvent): void {
     this.accordionItemToggled.emit(e);
   }
 
   render() {
+    const Header = () =>
+      h(
```

Comment:
`h()` is essentially [`React.createElement()`](https://reactjs.org/docs/react-api.html#createelement).